worker/update_device_status.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "update_device_status",
+    "description": "Update device status.",
+    "version": 1,
+    "tasks": [
+        {
+            "name": "update_device_identity_inventory",
+            "type": "http",
+            "http": {
+                "uri": "http://mender-inventory:8080/api/internal/v2/inventory/devices/${workflow.input.tenant_id}/${workflow.input.device_status}",
+                "method": "POST",
+                "contentType": "application/json",
+                "body": "${workflow.input.device_ids}",
+                "headers": {
+                    "X-MEN-RequestID": "${workflow.input.request_id}"
+                },
+                "connectionTimeOut": 1000,
+                "readTimeOut": 1000
```

Comment:
We increased  timeout to 8s in all the other workflowsa:
https://github.com/mendersoftware/workflows/blob/master/worker/decommission_device.json#L17
We also set the number of retries to 3:
https://github.com/mendersoftware/workflows/blob/master/worker/decommission_device.json#L9

Reply:
sure. fixed.